src/NHibernate/AdoNet/SqlClientBatchingBatcher.cs

```diff
@@ -70,6 +70,47 @@ public override void AddToBatch(IExpectation expectation)
 			}
 		}
 
+		public override Task AddToBatchAsync(IExpectation expectation, CancellationToken cancellationToken)
```

Comment:
As the target framework was changed to `netcoreapp2.0`, `SqlClientBatchingBatcher` is not compiled anymore and we have to manually add the async methods.

Reply:
So we should remove the partial keyword on this class.

Reply:
Removed.

Reply:
Off-topic considerations: Wondering on whether the type causing this to be NetFx only was still not available in core, I found out it seems it is available in netcoreapp2.x, but bugged and fixed only in 3.0. See dotnet/corefx#29391.
But for 5.0, we may have a batching API available, which would allow to remove most of our batcher implementations in favor of using that API. See dotnet/corefx#35135. (Assuming that targeting coreFx 5.0 implies dropping NetFx, as coreFx 5.0 is supposed to obsolete netFx.)